aws/main.tf

```diff
@@ -65,7 +65,7 @@ module "eks" {
       max_capacity     = 3
       min_capacity     = 1
 
-      instance_type = "t3.small"
+      instance_type = "t3.xlarge"
```

Comment:
I was seeing failures early on about sizing issues - it may work for core, but it's an issue for worker nodes.  Its easy enough to attempt it with small, but I'm sure that we'd hit a failure.

Reply:
There are separate node groups for user and core.  You're definitely right,  a t3.small is too small to do the actual notebook work and/or launch notebook sessions at all with the current user config in common.yaml.   But...  the core node group is something else.  I could imagine that failing due to autoscaling demands so I could see  this change happening,  but I just wanted to double check because it will add significant extra cost.

Reply:
If you believe that core can run on small, I can change it back.

Reply:
Switched to t3.small for now